components/Picture.tsx

```diff
@@ -0,0 +1,32 @@
+import styled from "styled-components";
+import Image from "next/image";
+
```

Comment:
Its best to add a comment for the reader here that you are working with typescript :))

Reply:
I will do that before the next push, thank you!